src/main/java/org/fogbowcloud/arrebol/execution/docker/request/WorkerDockerRequestHelper.java

```diff
@@ -105,7 +108,7 @@ private void setUpImage(TaskSpec taskSpec) {
         try {
             this.pullImage(image);
         } catch (Exception e) {
-            LOGGER.info("Error to pull docker image: " + image + " for the task spec " + taskSpec.getSpec() +
+            throw new NotFoundDockerImage("Error to pull docker image: " + image + " for the task spec " + taskSpec.getSpec() +
```

Comment:
In this case, you know the specific error that will be thrown. Change the phrase to something more cohesive, like `"Image " + image + " not found for the task spec " taskSpec.getSpec()`